tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -119,6 +119,10 @@ class WskBasicUsageTests
             wsk.action.get(fullQualifiedName).stdout should include(s"ok: got action ${packageName}/${actionName}")
     }
 
+    it should "include CLI user agent headers with outbound requests" in {
+        val stdout = wsk.action.list(verbose = true).stdout
+        stdout should include regex ("""\bUser-Agent\b": \[\s+"OpenWhisk\-CLI/1.\d+.*""")
```

Comment:
since the goal is just to run a cli command with verbose on, i'd rather you don't change the `list` method and install use `wsk.cli(Seq("list"), verbose = true)` directly.

Reply:
@rabbah - oh.. i'm wondering why the entity functions don't have a `verbose` pass through to the `wsk.cli` fuction?